external/java/src/main/java/ch/epfl/dedis/lib/darc/IdentityDarc.java

```diff
@@ -0,0 +1,96 @@
+package ch.epfl.dedis.lib.darc;
+
+import ch.epfl.dedis.lib.exception.CothorityCryptoException;
+import ch.epfl.dedis.proto.DarcProto;
+import com.google.protobuf.ByteString;
+
+import javax.xml.bind.DatatypeConverter;
+
+public class IdentityDarc implements Identity {
```

Comment:
Is this directly imported from `cothority`? If yes, you can leave it like that. If you wrote it, please add a class-comment here.